core/src/main/java/org/hisp/dhis/android/core/analytics/trackerlinelist/TrackerLineListRepository.kt

```diff
@@ -39,6 +39,8 @@ interface TrackerLineListRepository {
 
     fun withEnrollmentOutput(programId: String): TrackerLineListRepository
 
+    fun withTrackedEntityInstanceOutput(trackedEntityType: String): TrackerLineListRepository
```

Comment:
Maybe set `trackedEntityTypeId` as parameter name to keep consistency with `programId` and `programStageId`?